stonesoup/resampler/tests/test_particle.py

```diff
@@ -1,11 +1,26 @@
 import numpy as np
+import pytest
 
 from ...types.particle import Particle
 from ...types.state import ParticleState
-from ..particle import SystematicResampler
+from ..particle import SystematicResampler, MultinomialResampler, StratifiedResampler, \
+    ResidualResampler
 from ..particle import ESSResampler
 
 
+def test_raise_errors():
```

Comment:
Maybe could have a more informative name - sorry, getting nitpicky at this point. Tests are all looking good. Maybe change this to something like `test_residual_method_errors`.